xclim/core/dataflags.py

```diff
@@ -215,11 +234,15 @@ def temperature_extremely_low(
     Parameters
     ----------
     da : xarray.DataArray
+        The DataArray of temperatures being examined.
```

Comment:
might keep "being examined". After seeing all the xclim code, I see that a short descriptive comment if often added, so that's all right with me. Stylistically, I think we currently try to avoid starting the description with "The" in xclim? The prototype, simplest example beign: 

```python
da : xr.DataArray
    Input dataset.
```
and the "The input dataset"

Reply:
How do you feel about:
```python
da : xr.DataArray
    Input temperature DataArray.
```